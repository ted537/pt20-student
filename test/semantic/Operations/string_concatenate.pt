using input;
var str = "Hello" | "World";
