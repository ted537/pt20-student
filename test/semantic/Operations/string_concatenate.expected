.tFileDescriptor .tLiteralInteger .tFileBind .tLiteralAddress .tStoreInteger .tInitialValue .tSkipString .tStringData .tLiteralAddress .tFetchChar .tSkipString .tStringData .tLiteralAddress .tFetchChar .tConcatenate .tInitEnd .tLiteralAddress .tStoreChar .tTrapBegin .tTrap
